aea/helpers/profiling.py

```diff
@@ -194,6 +218,45 @@ def get_objects_instances(self) -> Dict:
             lock.release()
         return result
 
-    def get_objecst_created(self) -> Dict:
+    def get_objects_created(self) -> Dict:
         """Return dict with counted object instances created."""
         return self._counter
+
+
+def get_dialogues_by_type() -> Dict:
+    """Return dict with the number of dialogues by type."""
+
+    seen_dialogue_ids: List[int] = []
+    seen_dialogue_storage_ids: List[int] = []
+    dialogues_by_type: Dict[str, int] = {"incomplete_to_complete_dialogue_labels": 0}
+
+    lock.acquire()
+    try:
+        for obj in gc.get_objects():
+            if isinstance(obj, Dialogue) and id(obj) not in seen_dialogue_ids:
+                # Remember this dialogue
```

Comment:
This is suboptimal atm as we are iterating through the garbage collector twice, but I didn't want to mess with the profiling structure in this PR. Since we now have a ticket for redesigning the profiler, we will handle different tracked objects there.